waspc/cli/Command/Deps.hs

```diff
@@ -0,0 +1,27 @@
+module Command.Deps
+  ( deps,
+  )
+where
+
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import Generator.ServerGenerator as Server (waspNpmDeps)
+import qualified Generator.WebAppGenerator as WebApp (waspNpmDeps)
```

Comment:
Here you at least have `qualified`!
But it is still somewhat redundant all together, do the same thing as I suggested above for the ServerGenerator.